go/worker/compute/executor/committee/node.go

```diff
@@ -1423,6 +1456,11 @@ func NewNode(
 		return nil, err
 	}
 
+	cache, err := lru.New(lru.Capacity(lastScheduledCacheSize, false))
```

Comment:
If cache size is specified as zero this should probably disable the cache instead of making it unlimited (which sounds dangerous) which is what happens currently?